packages/toggle-model/test/Fixtures/Configuration.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pheature\Test\Model\Toggle\Fixtures;
+
+use Pheature\Model\Toggle\SegmentFactory;
+use Pheature\Model\Toggle\StrategyFactory;
+
+final class Configuration
+{
+    public static function create(): array
+    {
+        return [
+            'dependencies' => [
```

Comment:
Maybe these test configuration classes can be avoided by adding the config array as constants in the test classes? 
Or we are using each test configuration class in more than one test?

Reply:
done!